merchants/verkkokauppa/helpers.py

```diff
@@ -25,7 +25,17 @@
 def parse_datetime(string: str | None) -> datetime | None:
     if string is None:
         return None
-    return datetime.strptime(string, "%Y-%m-%dT%H:%M:%S.%f").astimezone(settings.VERKKOKAUPPA_TIMEZONE)
+
+    allowed_formats = [
+        "%Y-%m-%dT%H:%M:%S.%f",
+        "%Y-%m-%dT%H:%M:%S",  # Required due to Verkkokauppa API not returning milliseconds when seconds are zero.
+    ]
+    for fmt in allowed_formats:
+        try:
+            return datetime.strptime(string, fmt).astimezone(settings.VERKKOKAUPPA_TIMEZONE)
```

Comment:
Why not use [datetime.fromisoformat](https://docs.python.org/3/library/datetime.html#datetime.datetime.fromisoformat)?

Reply:
Fixed this to #1207